analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ComplianceAnalyzerPlugin.java

```diff
@@ -0,0 +1,278 @@
+                BufferedWriter br  = new BufferedWriter(fr);
+
+                // write the sed ouput
+                while ((s = stdInput.readLine()) != null) {
+                    br.write(s);
+                    br.newLine();
+                }
+
+                br.close();
+
+                // read any errors from the attempted command
+                System.out.println("Here is the standard error of the command (if any):\n");
+                while ((s = stdError.readLine()) != null) {
+                    System.out.println(s);
+                }
+            }  catch (IOException ex) {
+                logger.info(ex.toString());
+                ex.printStackTrace();
+            }
+        }
+
+        public void ApplyK8sJob() {
+            try {
+                // Deploy ConfigMap
+                V1ConfigMap configMap = Yaml.loadAs(new File("docker/master-config.yaml"), V1ConfigMap.class);
+                V1ConfigMap deployedConfigMap = new CoreV1Api().createNamespacedConfigMap(K8S_NAMESPACE, configMap, null, null, null);
+                System.out.println("Deployed ConfigMap: " + deployedConfigMap);
+
+                // Deploy Service
+                V1Service service = Yaml.loadAs(new File("docker/service.yaml"), V1Service.class);
+                V1Service deployedService = new CoreV1Api().createNamespacedService(K8S_NAMESPACE, service, null, null, null);
+                System.out.println("Deployed Service: " + deployedService);
+
+                // Define and apply a qmstr kubernetes job
+                Yaml.addModelMap("v1", "Job", V1Job.class);
+
+                File file = new File(PATCHED_JOB_FILE_NAME);
+                V1Job yamlJob = (V1Job) Yaml.load(file);
+                logger.info("Job definition: " + yamlJob);
+                BatchV1Api api = new BatchV1Api();
+
+                V1Job result = api.createNamespacedJob(K8S_NAMESPACE, yamlJob, null, null, null);
+                System.out.println(result);
+
+                // delete job
+                /*
+                V1Status deleteResult = api.deleteNamespacedJob(yamlJob.getMetadata().getName(),
+                KUBERNETES_NAMESPACE, null, null, null, null, null, new V1DeleteOptions());
+                logger.info(deleteResult);
+                s*/
+            } catch (IOException ex) {
+                logger.info(ex.toString());
+                logger.info("Could not find file: " + PATCHED_JOB_FILE_NAME);
+            } catch (ApiException ex) {
+                logger.error("Status code: " + ex.getCode());
+                logger.error("Reason: " + ex.getResponseBody());
+                logger.error("Response headers: " + ex.getResponseHeaders());
+                ex.printStackTrace();
+            }
+        }
+
+        @Override
+        public Optional<String> produce() {
+            // dump output for now
+            return Optional.of("License Compliance Plugin finished successfully");
```

Comment:
You can return Option.empty() and the server will just write a message with an empty payload

Reply:
Done in 3e9ca2e4.